config/management/src/verify.rs

```diff
@@ -41,50 +41,63 @@ pub struct Verify {
 
 impl Verify {
     pub fn execute(self) -> Result<String, Error> {
-        let storage: Storage = self.backend.backend.try_into()?;
-        storage
-            .available()
-            .map_err(|e| Error::LocalStorageUnavailable(e.to_string()))?;
+        let local_storage = self.backend.backend.create_storage(LocalStorage)?;
         let mut buffer = String::new();
 
         writeln!(buffer, "Data stored in SecureStorage:").unwrap();
         write_break(&mut buffer);
         writeln!(buffer, "Keys").unwrap();
         write_break(&mut buffer);
 
-        write_ed25519_key(&storage, &mut buffer, CONSENSUS_KEY);
-        write_x25519_key(&storage, &mut buffer, FULLNODE_NETWORK_KEY);
-        write_ed25519_key(&storage, &mut buffer, libra_global_constants::OWNER_KEY);
-        write_ed25519_key(&storage, &mut buffer, libra_global_constants::OPERATOR_KEY);
-        write_ed25519_key(&storage, &mut buffer, VALIDATOR_NETWORK_KEY);
+        write_ed25519_key(&local_storage, &mut buffer, CONSENSUS_KEY);
+        write_x25519_key(&local_storage, &mut buffer, FULLNODE_NETWORK_KEY);
+        write_ed25519_key(
+            &local_storage,
+            &mut buffer,
+            libra_global_constants::OWNER_KEY,
+        );
+        write_ed25519_key(
+            &local_storage,
+            &mut buffer,
+            libra_global_constants::OPERATOR_KEY,
```

Comment:
might be worth shoving this in the imports so we can return back to the single line :P